babel-watch.js

```diff
@@ -70,6 +71,7 @@ program.on('--help', () => {
   https://github.com/kmagiera/babel-watch
   `);
 });
+program.allowUnknownOption(true);
```

Comment:
@kmagiera Unknown options will be allowed, otherwise it will cause an error.
e.g. `babel-watch --harmony --no-warnings index.js`

`--harmony` and `--no-warnings` are unknown options, they will be passed to node process. Just like `node --harmony --no-warnings index.js`